workshops/templates/workshops/airport.html

```diff
@@ -11,7 +11,15 @@
 </table>
 
 <div class="clearfix">
+  {% if perms.workshops.change_airport %}
   <p class="edit-object pull-left"><a href="{% url 'airport_edit' airport.iata %}" class=" btn btn-primary">Edit</a></p>
+  {% else %}
+  <p class="edit-object pull-left"><a href="{% url 'airport_edit' airport.iata %}" class=" btn btn-primary disabled">Edit</a></p>
+  {% endif %}
```

Comment:
I'll add some other things you requested and get it back to you for final approval :)